sql/src/main/java/io/crate/analyze/relations/RelationSplitter.java

```diff
@@ -151,6 +150,7 @@ private void processWhere() {
         Symbol query = querySpec.where().query();
         assert query != null;
         QuerySplittingVisitor.Context context = QuerySplittingVisitor.INSTANCE.process(querySpec.where().query(), joinPairs);
+        WhereSymbolValidatorVisitor.INSTANCE.process(context.query(), null);
```

Comment:
if we do this in https://github.com/crate/crate/blob/1ccf42af49ebab80c35626249a97cef673b6445c/sql/src/main/java/io/crate/analyze/EvaluatingNormalizer.java#L116-L116 we do not require an additional visitor run.